feat: adds loading spinner for start page

dominikeinkemmer opened the pull request:
Resolves #168

**Proposed Changes**
Adds a loading spinner on the top page until we fetched the spinners from the database.

*Description of changes in this PR*
- Adds the vuetify `progress-circular` component as a loading spinner on the top page until the speakers are fetched from the db

**Testing Plan**

*How do the reviewers QA this? Feel free to write unit tests or describe manual testing steps here*
- Access start page and scroll down
- If results are already loaded you might have to refresh depending on how fast results are loading for you
   - One option to slow the db fetch down might be to simulate a slow mobile network via the chrome browser and clear the browser cache so the loading spinner shows for a longer time.

dominikeinkemmer commented on web/src/views/FindSpeaker.vue:
```diff
@@ -21,6 +21,17 @@
           </h2>
         </v-row>
+        <v-row
+          v-if="!speakers.length"
```
I think for now this condition is fine, but once the [pagination PR](https://github.com/WWCodeTokyo/speak-her-db/pull/166) is merged, maybe the condition should be changed a little bit so it shows on every pagination event 🤔

ann-kilzer replied:
Great, can you make this update?

dominikeinkemmer replied:
we need a new PR for moving the pagination into it's own component anyways right? how about fixing it together in that PR? :)

dominikeinkemmer replied:
either way is fine for me though, just let me know which you prefer :)

tuttiq commented:
Thanks @dominikeinkemmer ! 🎉  The current version looks good to me.

Since the pagination PR is merged, can you update this branch from master and see if it's easy to update the spinner to show when loading each page?

ann-kilzer reviewed (approved):
Looks fantastic, let's merge it with the pagination changes and then we can put it into prod.

dominikeinkemmer commented:
uum sorry something got messed up with the commits, I'll fix it 😅

dominikeinkemmer commented on web/src/components/Pagination.vue:
```diff
@@ -37,7 +37,7 @@ export default {
   computed: {
     isFirstPage() {
-      return this.$props.pageStart <= 1;
+      return this.$props.firstEntry <= 1;
```
just noticed this was broken as well 🙇

dominikeinkemmer commented:
@tuttiq @ann-kilzer sorry something got messed up with the commits so I rebased and force pushed. I added the small change to the condition for rendering this and fixed one small bug I found :)

ann-kilzer commented:
Try not to force-push going forward, as it wrecks the comment history in github and makes it harder for others to work with your branch. I wonder if clicking the button to merge with master here made things difficult for you? I'll keep an eye on that going forward and see if there's a better way.

I kicked off a Netlify preview and will check back tomorrow morning as it is 🌃

ann-kilzer reviewed (approved):
LGTM 🌀

dominikeinkemmer commented:
ah yeah, the reason I force pushed this time was because the whole commit history got somehow messed up after I merged upstream master into my branch so I wanted to overwrite it all with a clean commit as my branch had 300+ changes all of a sudden 😅

ann-kilzer commented:
Okay, try merging going forward rather than rebasing / force pushing. See our [contributor guidelines](https://github.com/WWCodeTokyo/speak-her-db/blob/master/CONTRIBUTING.md#using-git).
